sections/04_study.md

```diff
@@ -52,7 +52,104 @@ particularly notable in this area?*
+accurate contact prediction. Evolutionary coupling analysis (ECA) is an 
+effective contact prediction method for some proteins with a very large 
+number (>1000) of sequence homologs, but ECA fares poorly for proteins 
+without many sequence homologs. Since (soluble) proteins with many 
+sequence homologs are very likely to have a good template in PDB, to make 
+contact-assisted folding practically useful for ab initio folding, it is essential 
+to predict accurate contacts for proteins without many sequence homologs. 
+By combining ECA with a few other protein features, shallow neural 
+network-based methods such as MetaPSICOV [@doi: 
+10.1093/bioinformatics/btu791] and CoinDCA-
+NN[@doi:10.1093/bioinformatics/btv472] have shown some advantage 
+over ECA for proteins with a small number of sequence homologs, but their 
+accuracy is still not very good. In recent years, deep learning methods have 
+been explored for contact prediction. For example, Di Lena et al. introduced 
+a deep spatio-temporal neural network (up to 100 layers) that utilizes both 
+spatial and temporal features to predict protein 
+contacts[@doi:10.1093/bioinformatics/bts475]. Cheng group combined 
+deep belief networks and boosting techniques to predict protein contacts 
+[@10.1093/bioinformatics/bts598] and trained deep networks by layer-wise 
+unsupervised learning followed by fine-tuning of the entire network. 
+Elofsson group developed an iterative deep learning technique for contact 
+prediction, in which Random Forests are applied to predict contacts at each 
+iteration [@doi:10.1371/journal.pcbi.1003889]. However, blindly tested in 
+the well-known CASP competitions, these methods did not show any 
+advantage over MetaPSICOV[@doi: 10.1093/bioinformatics/btu791], a 2-
+layer neural network method. Only until 2016, Xu group proposed a novel 
+deep learning method [@doi:10.1371/journal.pcbi.1005324] that can 
+significantly improve contact prediction accuracy over MetaPSICOV 
+especially for proteins without many sequence homologs. Xu’s deep model 
+is formed by one 1D residual neural network and one 2D residual neural 
+network. Blindly tested in the latest CASP competition (i.e., CASP12), Xu’s 
+deep learning method is ranked first in terms of the total F1 score (a widely-
+used performance metric) on free-modeling targets as well as the whole set 
+of targets. In this test, the group ranked second also employed a deep 
+learning method. Even MetaPSICOV, which ranked third, employed deeper 
+and wider layers than its old version. Xu group has also demonstrated in 
+another blind test CAMEO (which can be interpreted as a fully-automated 
+CASP) [@url:http://www.cameo3d.org/] that the predicted contacts can 
+fold quite a few proteins with a novel fold and only 65-330 sequence 
+homologs. Xu’s method also works well on membrane protein contact 
+prediction even if trained mostly by non-membrane proteins.
```

Comment:
I guess it is possible to further improve. For example, we may try different network architectures. Data is the bottleneck. Some proteins just do not have any sequence homologs, which may fail any methods.

In terms of application, it really depends. For some proteins, we can produce really good models even by our contact-assisted folding. For example, a few weeks ago in the blind CAMEO test, our contact-based web server predicted 3D models for two membrane proteins of >200 residues with RMSD close to 2 Angstrom. Many proteins can also be well modeled by template-based methods. However, there are still many proteins for which we cannot produce high-resolution models. Nevertheless, for protein-protein interaction prediction, we may not need a very high resolution.